rails/locale/az.yml

```diff
@@ -4,6 +4,9 @@ az:
     errors:
       messages:
         record_invalid: 'Yoxlama uğursuz oldu: %{errors}'
+        restrict_dependent_destroy:
+          has_one: "Əlaqəli %{record} yazı olduğundan bu yazını silmək mümkün deyil"
+          has_many: "Əlaqəli %{record} yazılar olduğundan bu yazını silmək mümkün deyil"
```

Comment:
@NARKOZ none of turkic languages, including azerbaijani, accept pluralization suffix if already prefixed with numbers. Ex: 1 yazı, (_en: one record_), 23 yazı (_twenty three record**s**_),  42 yazı (_fourty two record**s**_)